src/log_windows.cpp

```diff
@@ -84,7 +84,7 @@ console_handler::console_handler()
 		// already pointing to the console.
 		LOG_LS << "Console already attached at startup (built with console subsystem flag?), log file disabled.";
 	} else {
-		enable_native_console_output();
+		enable_native_console_output(no_con);
```

Comment:
Is there a point calling this function at all if no_con is true?

Reply:
Sure but what is the point?

Reply:
It's at least an indication that *something* happened, I suppose.

Reply:
I don't really understand. What is happening if you attach or allocate a console and then don't redirect to it? Presumably nothing useful? We didn't do it before so why should we do it now?

Reply:
Updated.

Reply:
Thanks.